11/19/21 16:20:34,de,https://www.hornbach.de/cms/de/de/sortiment/bad-sanitaer.html,"HTTPSConnectionPool(host='www.hornbach.de', port=443): Max retries exceeded with url: /cms/de/de/sortiment/bad-sanitaer.html (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000022CC79F7580>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))on line:81"
11/19/21 16:20:37,de,https://www.hornbach.de/cms/de/de/sortiment/bad-sanitaer.html,"HTTPSConnectionPool(host='www.hornbach.de', port=443): Max retries exceeded with url: /cms/de/de/sortiment/bad-sanitaer.html (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000022CC79F7730>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))on line:81"
11/19/21 16:20:40,de,https://www.hornbach.de/cms/de/de/sortiment/bad-sanitaer.html,"HTTPSConnectionPool(host='www.hornbach.de', port=443): Max retries exceeded with url: /cms/de/de/sortiment/bad-sanitaer.html (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000022CC79F7310>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))on line:81"
